src/core/lib/surface/call.c

```diff
@@ -549,8 +549,8 @@ static void destroy_call(grpc_exec_ctx *exec_ctx, void *call,
     GRPC_CQ_INTERNAL_UNREF(exec_ctx, c->cq, "bind");
   }
 
-  get_final_status(call, set_status_value_directly, &c->final_info.final_status,
-                   NULL);
+  get_final_status((grpc_call *)call, set_status_value_directly,
```

Comment:
Wait, this has already been typecasted to a call as `c` above (in line 527)

Reply:
Yes. Good eye!